src/utils/enum_helper.h

```diff
@@ -66,6 +66,11 @@
 
 #define ENUM_END(type) ENUM_END2(type, type)
 
+#define ENUM_CONST(str) k##str
+#define ENUM_CONST_DEF(str) ENUM_CONST(str),
+#define ENUM_CONST_REG_STR(enum_class, str)                                                        \
+    helper->register_enum(#str, enum_class::ENUM_CONST(str));
```

Comment:
Could you please add some tests to check the enum_helper_xxx works well as expect? Use the existing `metric_type` is OK.

Reply:
OK, I'll add some test cases.